build_runner_core/pubspec.yaml

```diff
@@ -39,9 +39,5 @@ dev_dependencies:
     path: ../_test_common
 
 dependency_overrides:
-  build:
-    path: ../build
-  build_resolvers:
-    path: ../build_resolvers
   # https://github.com/dart-lang/sdk/issues/38499
   analyzer: 0.38.2
```

Comment:
I think you can't publish with a dependency_override like this? Are you planning on removing locally during the publish?

Reply:
Yes I am removing locally before publish